### ISSUE TITLE
Style/TrailingUnderscoreVariable should give the user a valid recommendation.

### ISSUE THREAD

**Issue by envygeeks** [bbatsov/rubocop#2660]
At this time doing `_, e, _` will result in a violation of the aforementioned cop without guiding them to doing Ruby the right way... as in: `_, e,`, this could be because of something else but ultimately normally we do the latter rather than the former, but recently did the former and it triggered that error and confused even me for a second while I was trying to figure out what it was on about.

**Comment by alexdowad** [comment]
Sorry, @envygeeks, can you please post a snippet of valid Ruby code which can be used to reproduce the problem? It would also be helpful if you can show both 1) what the current output of RC is and 2) what you suggest it should be.

**Comment by envygeeks** [comment]
```ruby
class MyClass
  def initialize
    @a, _, _, _ = test
  end

  def test
    %W(
      shit just got real
    )
  end
end
```

```
test.rb:4:9: C: Do not use trailing _s in parallel assignment.
    @a, _, _, _ = test
        ^^^^^^^
```

The error gives no hint that `@a, = test` is the better way.  I think the error would be better as:

```
test.rb:4:9: C: Do not use trailing _s in parallel assignment, prefer `_, var, = blah`
    @a, _, _, _ = test
        ^^^^^^^
```

**Comment by envygeeks** [comment]
Actually, I don't care what improvement is made (probably) as long as it's not so vague.

**Comment by alexdowad** [comment]
> prefer `_, var, = blah`

Agreed, but better still, it should use the name of the actual variable from the code here rather than `var`.
